### DISCUSSION TITLE
Request for new feature: Seria possível adição de suporte ao ICS/Uminho?

### DISCUSSION THREAD

**Discussion by leoneldca** [joaomlourenco/novathesis#168] [Ideas]
Olá João, 
Parabéns pelo seu trabalho!

Seria possível a adição do  ICS/Uminho?
Instituto de Ciências Sociais, Doutoramento em Estudos Culturais

Segue abaixo o que tenho de informações:

Normas para formatação de teses e dissertações, Uminho
[https://alunos.uminho.pt/PT/estudantes/Paginas/InfoUteisFormatacao.aspx](https://alunos.uminho.pt/PT/estudantes/Paginas/InfoUteisFormatacao.aspx)

Página sobre a identidade visual do Instituto:
https://www.ics.uminho.pt/pt/Comunicacao/Paginas/Identidade-Visual.aspx

Logomarca
https://www.ics.uminho.pt/pt/Comunicacao/Documents/ICS.zip

Fonte ( A mesma pra toda a Uminho)
https://www.ics.uminho.pt/pt/Comunicacao/Documents/NewsGotT.zip

Desde já agreço imensamente pela atenção

Atenciosamente

Leonel Diógenes Carvalhaes Alvarenga

**Comment by b-pereira** [comment]
@leoneldca  todas as normas de formatação da UMINHO já estão incluídas junto do _template_ para a Escola de Engenharia. Seria apenas mudar o logótipo para o ICS, nome do Instituto. O resto é configurável no _template_. No meu [fork](https://github.com/b-pereira/novathesis) eu consigo fazer essas alterações bem rápido. Eu pensei na altura criar para todas as escolas e institutos da UMINHO, mas o habitual atrito por pessoas que não das áreas Ciências/Engenharias/Matemática, devido às dificuldades com o LaTeX e preferência pelo Word, fizeram-me mudar de ideias. 

Segundo o que sei, o Prof. @joaomlourenco  ainda tem que arranjar forma de organizar forma de organizar o código do _template_ da UMINHO para não duplicá-lo (existe a ideia de adicionar o _template_ para a Escola de Medicina e Escola de Ciências, assim como tenho o meu). Se for urgente e quiseres usar o meu, posso fazer isso. @leoneldca que dizes?

**Comment by joaomlourenco** [comment]
@b-pereira sem querer desvalorizar o teu trabalho e colaboração, nesta altura “investir” em escrever a tese com o tempalte v4 é pedir problemas, pois será bastante mais difícil obter suporte.  Espero publicar uma versão revista que suportará facilmente qualquer escola da uminho até ao final deste mês de Abril (se der para esperar até lá parece-me mais sensato).

@b-pereira assim que isto estiver preparado para "receber" as várias escolas da UMinho, delego em ti a tarefa de as "criar" a todas… pode ser? 👍

**Reply by b-pereira** [reply]
@joaomlourenco eu já poderia ter feito isso até para a EMUM e ECUM, se não fosse não saber muito bem como reutilizar o código. 🤔 Vou ver isso

**Comment by joaomlourenco** [comment]
The current version (6.3.0) supports **ALL** the Schools from UMinho.
Please fill an issue/bug report if you find bugs.  Thanks.

**Reply by leoneldca** [reply]
Obrigado a todos pela atenção com minha solilcitação. Tenho certeza que será de grande valia pra muitos estudantes.
Parabéns também a todos pela dedicação com o projeto, em todas as frentes de trabalho. @joaomlourenco @b-pereira
Boa semana e bom trabalho.!!

**Comment by joaomlourenco** [comment]
There is full suport for UMinho now!
